mergify_engine/rules/__init__.py

```diff
@@ -555,6 +555,14 @@ def FullifyPullRequestRules(v):
     return v
 
 
+CommandsRestrictionsSchema = {
+    voluptuous.Required("conditions", default=[]): voluptuous.All(
```

Comment:
Do we allow `None` to allow `conditions:`?

Reply:
Hum, if it's empty the whole section is useless, so.